docroot/themes/custom/mass_admin_theme/css/components/paragraphs.admin.css

```diff
@@ -76,3 +76,16 @@ div.field--widget-entity-reference-paragraphs .description > strong {
 .field--widget-entity-reference-paragraphs .form-radios label {
   word-break: break-word;
 }
+
+@media screen and (max-width: 1120px) {
+  table#field-org-ref-locations-values tr.draggable td + td {
+    display: block;
```

Comment:
```suggestion
    min-width: 20rem;
```

Reply:
setting a min-width to display more of the input text when on its own row
![Uploading Screen Shot 2021-11-29 at 3.52.19 PM.png…]()